.lintrunner.toml

```diff
@@ -253,6 +254,16 @@ exclude_patterns = [
     '**/*pb.h',
     '**/*CUDA*',
     '**/cuda/*pp',
+    'c10/util/complex_math.h',
+    'c10/util/complex_utils.h',
+    'c10/util/flat_hash_map.h',
```

Comment:
This can be removed from here since you fixed it right?

Reply:
They are either internal headers or third-party headers so I prefer to exclude them.